src/util/account_util.js

```diff
@@ -283,11 +283,11 @@ function create_access_key_auth(req, account, is_iam) {
             account.access_keys = access_keys;
         }
         decrypted_access_keys = _.cloneDeep(account.access_keys);
-        account.access_keys[0] = {
-            access_key: account.access_keys[0].access_key,
```

Comment:
I think we dont need to change here, this method is getting called from `create_account()` and access key always will be first item in that array.

Reply:
Sure, Thanks